internal/rbd/rbd_util.go

```diff
@@ -1934,6 +1934,24 @@ func (rv *rbdVolume) setVolumeMetadata(parameters map[string]string) error {
 	return nil
 }
 
+// unsetVolumeMetadata unset PV/PVC/PVCNamespace metadata on RBD image.
+func (rv *rbdVolume) unsetVolumeMetadata() error {
```

Comment:
in order to keep aligned with the parameters passed to `setVolumeMetadata()`, it would be better to pass a `[]string` with the keys instead of fetching them inside the function

Reply:
it is out of balance with `setVolumeMetadata()`, so I prefer that you pass the keys. Possibly other metadata gets added later on, which does not get retrieved from the k8s API.

Reply:
Indeed! additional metadata should be passed as an argument to `setVolumeMetadata()`, so the caller of `unsetVolumeMetadata()` should have that as well. The `rbdVolume` functions are an API for external callers, and there should not be a dependency on hard-coded metadata keys that are not internal to the `rbdVolume`.

Reply:
@nixpanic Done now, PTAL